src/webapp/js/components/health-workers-form.js

```diff
@@ -8,9 +8,8 @@ import DateTime from 'react-datetime';
 import 'react-datetime/css/react-datetime.css';
 
 import FormField from './form-field';
-import { createHealthWorker } from '../actions';
-import { clearFields, getAttributesForObjectSelect } from '../utils/form-utils';
-import apiClient from '../utils/api-client';
+import { createHealthWorker, fetchLocations } from '../actions';
```

Comment:
I think you can delete createHealthWorker from here. Please let me know.

Reply:
yes, you're right, I forgot to remove it during previous refactor